README.md

```diff
@@ -48,17 +60,19 @@ Now you're ready to do something awesome!
 
 
 <!-- ROADMAP -->
-## Roadmap
```

Comment:
add a single <br /> above this & change the scaling to 0.5x i.e. `##` from `###`